packages/pulpcore-obsolete-packages/pulpcore-obsolete-packages.spec

```
@@ -1,14 +1,14 @@
 Name: pulpcore-obsolete-packages
 Version: 1.0
-Release: 1%{?dist}
+Release: 2%{?dist}
 License: MIT
 Summary: A package to obsolete retired packages
 URL: https://github.com/theforeman/pulpcore-packaging
 BuildArch: noarch
 
-Obsoletes:      python3-django-currentuser
+Obsoletes:      python3-django-currentuser <= 0.5.3
```

```suggestion
Obsoletes:      python3-django-currentuser < 0.5.3-6
```

The last build was 0.5.3-5, so we should make it like this :)